benchmark/nixlbench/contrib/Dockerfile

```
@@ -26,6 +26,21 @@ FROM ${BASE_IMAGE}:${BASE_IMAGE_TAG} AS os_setup_stage
 # Re-declare for use in this stage
 ARG ARCH="x86_64"
 ARG DEFAULT_PYTHON_VERSION
+ARG APT_MIRROR=""
+
+# Optional apt mirror override, e.g. for CI environments with an internal
+# Ubuntu mirror. Empty (default) keeps the public archive/ports.ubuntu.com.
+RUN if [ -n "$APT_MIRROR" ]; then \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)